src/form-styles/form-styles-multi-step.ts

```diff
@@ -403,14 +403,14 @@ export class formStylesMultiStep  {
         `;
     }
 
-    private addStyles(): void {
+    addStyles(): any {
```

Comment:
o ile to mozliwe to unikamy zwracania : any jako typu. Wiec jezeli jestes w stanie zdeterminowac typ to to zrobmy